Source/Core/Core/AchievementManager.h

```diff
@@ -58,11 +63,13 @@ class AchievementManager
   static constexpr size_t RP_SIZE = 256;
   using RichPresence = std::array<char, RP_SIZE>;
   using Badge = std::vector<u8>;
+  static constexpr u32 BADGE_HW = 96;
 
   struct BadgeStatus
   {
     std::string name = "";
     Badge badge{};
+    std::shared_ptr<OSD::Icon> osd_icon;
```

Comment:
Is there a noticeable performance advantage to decoding these ahead of time rather than on-demand or what are you trying to accomplish with holding on to a second copy of what is effectively already stored in `badge`?